vars/perfRegressionParallelPipelinebyRegion.groovy

```diff
@@ -45,29 +45,29 @@ def call(Map pipelineParams) {
                             [
                                 job_name: 'scylla-enterprise/perf-regression/scylla-enterprise-perf-regression-predefined-throughput-steps-vnodes',
                                 region: 'us-east-1',
-                                versions: ['2024.1', '2024.2', '2025.1', '2025.2', 'master'],
+                                versions: ['2024.1', '2024.2', '2025.1', '2025.2', '2025.3', '2025.3', 'master'],
```

Comment:
also maybe we should list now 2025.4 in all ? (buy us a bit more time to fix it properly ?)